docs/overview/learning-path-beginner.md

```diff
@@ -51,6 +51,10 @@ Then return to Script Lab and use it like a playground: make your own code chang
 
 Get an understanding of the purposes of the add-in manifest and an introduction to its XML markup or JSON in [Office Add-ins manifest](../develop/add-in-manifests.md).
 
+## Step 7: Create a Partner Center account
+
+Create a [Partner Center account](/partner-center/marketplace-offers/open-a-developer-account). This could take some time. If you plan to [publish your add-in to AppSource](../publish/publish.md), get this process going as soon as possible.
```

Comment:
Since this is only for folks publishing to AppSource, consider leading with that. Feel free to rephrase however (and make the same change in the other spot).
```suggestion
If you plan to [publish your add-in to AppSource](../publish/publish.md), create a [Partner Center account](/partner-center/marketplace-offers/open-a-developer-account). This could take some time. Get this process going as soon as possible to avoid release delays.
```